Bugfix/bsc#1051838

lproven opened the pull request:
In a comment on Bugzilla, Thorsten Kukuk asked for an example using a password hash rather than plaintext.

This also makes the separate worker-node and admin-node example files more useful.

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -861,21 +861,19 @@
-       and
-      </para>
-      <screen>~/cloud-config/cc-admin/meta-data</screen>
+      <screen>
```
no line break

lproven replied:
Done.

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -861,21 +861,19 @@
-      <screen>~/cloud-config/cc-admin/meta-data</screen>
+      <screen>
+~/cloud-config/cc-admin/user-data
+~/cloud-config/cc-admin/meta-data
```
no line break

lproven replied:
Done.

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -861,21 +861,19 @@
-       and
-      </para>
-      <screen>~/cloud-config/cc-worker/meta-data</screen>
+      <screen>
```
no line break

lproven replied:
Done.

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -861,21 +861,19 @@
-      <screen>~/cloud-config/cc-worker/meta-data</screen>
+      <screen>
+~/cloud-config/cc-worker/user-data
+~/cloud-config/cc-worker/meta-data
```
no line break

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -899,10 +897,11 @@ network-interfaces: |
 #cloud-config
 debug: True
 disable_root: False
+ssh_deletekeys: False
```
Care to elaborate?

lproven replied:
No -- I was taking the example files from Bugzilla. 

Googling suggests that "for public instances this should definitely be 'true'". Happy to change it if you think that is preferable.

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -899,10 +897,11 @@ network-interfaces: |
 chpasswd:
   list: |
-    root:<replaceable>root</replaceable>
+    root:<replaceable>MyPassword</replaceable>
```
MY_PASSWORD (replacables are written in caps)

lproven replied:
Done.

cwickert replied:
The team hasn't agreed on a solution yet.

lproven replied:
OK, fair enough. Are you happy with my amendments to this at present, though, or do you want more changes?

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -921,10 +920,11 @@ suse_caasp:
 chpasswd:
   list: |
-    root:<replaceable>root</replaceable>
+    root:<replaceable>MyPassword</replaceable>
```
MY_PASSWORD

lproven replied:
I can only find one instance of "MyPassword".

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -936,20 +936,28 @@ suse_caasp:
+       15, substituting the name of the folder containing the configuration
+       files for a &worker_node; and titling the volume <literal>cc-worker</literal>:
       </para>
+      <screen>
```
no line break

cwickert commented on xml/deployment_installation.xml:
```diff
@@ -936,20 +936,28 @@ suse_caasp:
+       files for a &worker_node; and titling the volume <literal>cc-worker</literal>:
       </para>
+      <screen>
+&prompt.user; sudo mkisofs -output <replaceable>cc-worker.img</replaceable> -volid <replaceable>cidata</replaceable> -joliet -rock <replaceable>cc-worker</replaceable>
```
no whitespace after &prompt.user, it's already part of the entity.
tag command as <command>.
wrap the command line with \ if it doesn't fit into a single line in the pdf version.
no line break before the end of the screen

lproven commented:
I think I have addressed all your comments now. If there are any outstanding, please tell me.

The build failure seems to be due to some external SLE material but I am not experienced at interpreting Travis output.
